action.yml

```diff
@@ -18,47 +18,28 @@ inputs:
     required: false
     default: ""
   dispatch_type:
-    description: "'workflow' or 'repository' (repository dispatch is deprecated and will be removed)"
+    description: "'workflow' or 'repository' (dispatch_type is deprecated and will be removed)"
```

Comment:
I don't see that `dispatch_type` is used in the action. Why do we keep it?
If a control sends an argument not in the input argument list, will the action fail?

Reply:
we will get a warning, for example:
> Warning: Unexpected input(s) 'dispatch_type', valid inputs are ['docker_user', 'docker_password', 'security_control', 'security_control_args', 'security_control_output_file']